scripts/actions/fetch-and-deserialize.js

```diff
@@ -40,15 +40,20 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
   const zipEntries = zip.getEntries();
 
   const translatedHtml = zipEntries.map((entry) => {
+    console.log(entry.entryName);
     return {
-      path: entry.entryName.split('/').pop().split('.').slice(0, -1), // get the last element from the folder
+      path: entry.entryName
```

Comment:
Its a bit hard to follow what this is doing. This also feels like it could be a touch brittle. I would suggest a couple changes.

1) Since you're trying to get rid of `src/jp` as part of the name, I would suggest just using a `replace`. Its a bit clearer the intention and harder to break:

```js
entry.entryName.replace(`src/${locale}`, '')
```

2) Use `path.basename` + `path.extname` to get rid of the extension. This makes it a bit clearer and ensures edge cases are handled:

```js
path.basename(entry.entryName, path.extname(path.entryName)) // => index.mdx -> index
```

Putting that altogether makes it easier to understand the purpose:

```js
const path = require('path')

const filepath = entry.entryName.replace(`src/${locale}`)

return {
  path: path.basename(filepath, path.extname(filepath))
}
```

Reply:
yes! I like this suggestion :)